src/plugin/arc/query/arcqueryhandler.js

```diff
@@ -69,7 +69,7 @@ plugin.arc.query.ArcQueryHandler.prototype.getActiveEntries = function() {
   }
 
   // ignore disabled areas
-  entries = entries.filter(os.ui.query.QueryHandler.shownAreas);
+  entries = entries.filter(os.ui.query.QueryHandler.shownAreas.bind(null, null));
```

Comment:
With the manager parameter second, you'll still need to use an arrow function because `filter` passes `item, index, array` params to the callback. So these will be:
```
entries = entries.filter((entry) => os.ui.query.QueryHandler.includes(entry));
```

Reply:
Ahh, that is exactly what I was looking for - didn't want this be a breaking change but couldn't quite find a way around that